src/main/java/org/folio/circulation/domain/OpeningDay.java

```diff
@@ -34,15 +37,23 @@ static OpeningDay createClosedDay() {
   private boolean allDay;
   private boolean open;
 
-  OpeningDay(JsonObject jsonObject, String key) {
-    JsonObject openingDayJson = jsonObject.getJsonObject(key);
+  OpeningDay(JsonObject openingDayJson) {
+    requireNonNull(openingDayJson, "Json object cannot be null");
+    this.allDay = openingDayJson.getBoolean(ALL_DAY_KEY, false);
```

Comment:
Personally, I would introduce a blank line between the guard clause and the rest of the logic.

Reply:
done